go-apps/meep-auth-svc/server/auth-svc.go

```diff
@@ -407,18 +435,33 @@ func cacheServicePermissions(cfg *PermissionsConfig) {
 			// Use cache default permission if service-specific default is not found
 			permission = authSvc.cache.Default
 		}
-		svcMap[svc.Name] = permission
 
-		// Add auth service default route
-		route := new(AuthRoute)
-		route.Name = svc.Name
-		route.Prefix = true
+		// Add auth service routes + cache service permissions
 		if svc.Sbox {
+			// Mep-specific sandbox service
+			route := new(AuthRoute)
+			route.Prefix = true
+			route.Name = mepPrefix + apiPrefix + svc.Name
+			route.Pattern = "/{sbox}/{mep}" + svc.Path
+			routes = append(routes, route)
+			svcMap[route.Name] = permission
+
+			// Sandbox service
+			route = new(AuthRoute)
+			route.Prefix = true
+			route.Name = apiPrefix + svc.Name
 			route.Pattern = "/{sbox}" + svc.Path
+			routes = append(routes, route)
+			svcMap[route.Name] = permission
 		} else {
+			// Global service
```

Comment:
same as previous comment

Reply:
Agreed. Will try to address when working in this module again.